packages/fxa-settings/src/components/AppLayout/index.tsx

```diff
@@ -20,25 +21,31 @@ export const AppLayout = ({
   primaryEmail,
   hasSubscription,
   children,
-}: AppLayoutProps) => (
-  <div className="flex flex-col min-h-screen" data-testid="app">
-    <ContentSkip />
-    <HeaderLockup
-      {...{
-        avatarUrl,
-        primaryEmail,
-      }}
-    />
-    <div className="max-w-screen-desktopXl w-full mx-auto flex flex-1 tablet:px-20 desktop:px-12">
-      <div className="hidden desktop:block desktop:flex-2">
-        <Nav {...{ hasSubscription, primaryEmail }} />
+}: AppLayoutProps) => {
+  const alertBarRootRef = useRef<HTMLDivElement>(null);
+
+  return (
+    <div className="flex flex-col min-h-screen" data-testid="app">
+      <ContentSkip />
+      <HeaderLockup
+        {...{
+          avatarUrl,
+          primaryEmail,
+        }}
+      />
+      <div className="max-w-screen-desktopXl w-full mx-auto flex flex-1 tablet:px-20 desktop:px-12">
+        <div className="hidden desktop:block desktop:flex-2">
+          <Nav {...{ hasSubscription, primaryEmail }} />
+        </div>
+        <main id="main" data-testid="main" className="desktop:flex-7">
+          <div id="alert-bar-root" data-testid="alert-bar-root" ref={alertBarRootRef} />
```

Comment:
Needs a closing `</div>` 😜

Reply:
Per our Slack chat, JSX is cool with this syntax. Kinda feels gross to be writing invalid HTML to me, but it's fully supported and I'm just a crank, so you can ignore this and the last comment!

Reply:
Lol but JSX is not HTML, `className` isn't a thing ;)

Reply:
_Throws gloves on the ground_

**Well actually, I—**